cogs/instances.py

```diff
@@ -584,12 +584,15 @@ async def instance_config(self, ctx, key: str = None, value = None):
                         raise commands.BadArgument('You haven\'t joined that guild yourself')
                     if not member.guild_permissions.administrator:
                         raise commands.BadArgument('You need to have administrator permissions in that guild')
-
+                if key == "chat_trigger_words":
+                    if value[-1] == ',':
+                        # Removing trailing , if inserted by mistake results in all chat to sent to alert channel
+                        value = value.rstrip(',')
```

Comment:
Can just use `value.strip(',')` here. The if statement is redundant but doesn't hurt I guess.